### ISSUE TITLE
The plugin file, doesn't have a .jar file.

### ISSUE THREAD

**Issue by Azm0deuz** [isidorjkaplan/UltraPlugin#2]
Without a .jar file the server can't even runs the library let alone read them, is their any chance we could get that fixed so people could use it.

**Comment by isidorjkaplan** [comment]
Indeed, this is just the raw code. It must be compiled in order to get the jar file. I'll look if I have one on hand to upload.
